internal/logger/log.go

```diff
@@ -6,7 +6,7 @@ SPDX-License-Identifier: AGPL-3.0-only
 
 /*
 Package logger provides logging functionality for Constellation services.
-It is a thin wrapper around the zap package, providing a consistent interface for logging.
+It is a thin wrapper around the slog package, providing a consistent interface for logging.
```

Comment:
Wasn't the purpose of the ticket to get rid of the wrapper and use slog directly?

Reply:
cc @miampf 

After internal discussion, we concluded that we rather want to get rid of the wrapper entirely and replace the logging calls we do in our code with using slog directly, feel free to consult me before implementation or if unsure anywhere. This was a misunderstanding and I didn't explain the issue to you properly - sorry for that. However, pivoting here does not mean that the work you did so far is worthless. You should be able to reuse almost everything you have so far.